android/app/src/main/java/com/microsoft/codepush/common/connection/QueryUpdateJob.java

```diff
@@ -0,0 +1,90 @@
+package com.microsoft.codepush.common.connection;
+
+import com.microsoft.appcenter.utils.AppCenterLog;
+import com.microsoft.codepush.common.datacontracts.CodePushUpdateResponse;
+import com.microsoft.codepush.common.exceptions.CodePushFinalizeException;
+import com.microsoft.codepush.common.exceptions.CodePushQueryUpdateException;
+import com.microsoft.codepush.common.utils.CodePushUtils;
+import com.microsoft.codepush.common.utils.FileUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.util.Arrays;
+import java.util.Scanner;
+
+import javax.net.ssl.HttpsURLConnection;
+
+import static com.microsoft.codepush.common.CodePush.LOG_TAG;
+
+/**
+ * Performs sending status reports to server.
+ */
+public class QueryUpdateJob extends BaseHttpJob<CodePushUpdateResponse> {
```

Comment:
I would prefer to use the same terms as we've been using before, instead of `QueryUpdateJob` we can use `CheckForUpdateJob`, `RepostStatusDownloadJob`, `RepostStatusDeployJob`, it looks consistent and more clear in terms of existing ecosystem.

Reply:
@sergey-akhalkov We have only one `ReportStatusJob` instead of 2.